projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -816,6 +816,17 @@ function prepare_for_akismet( $form ) {
 			}
 		}
 
+		/**
+		 * Filter the values that are sent to Akismet for the spam check.
+		 *
+		 * @module contact-form
+		 *
+		 * @since 10.2
+		 *
+		 * @param array $form The form values being sent to Akismet.
+		 */
+		$form = apply_filters( 'jetpack_contact_form_akismet_values', $form );
+		
 		return $form;
```

Comment:
I think you could potentially put the filter on the return directly:

```suggestion
		return apply_filters( 'jetpack_contact_form_akismet_values', $form );
```